lib/usecases/applicationDetails.js

```diff
@@ -65,11 +65,17 @@ export default async ({ clientGeneratedId }) => {
         councilRentAccountNumber: application.council_rent_account_number,
         councilTaxNumber: application.council_tax_number,
         businessAddress: {
-          uprn: application.business_address_uprn,
+          uprn:
+            application.business_address_uprn === '0000000'
```

Comment:
A bit of me dies inside when I see empty strings.
I wonder if we should leave it as it is? As finding the manual ones will need us to search for both '0000000' and empty string.
WDYT?

Reply:
I know, the cleanest way would be having `uprn` and `postcode` not mandatory and remove the fake values we stored in the DB. This is a quick (hacky) win 😅

Reply:
Instead of `''` maybe `'(Manually entered)'`?

Reply:
Or `'(Entered manually)'`

Reply:
@WillGibson the problem here is that we hardcode the value when we save it, it was a terrible hack made a couple of hours after going live 🙈